of-builder/main.go

```diff
@@ -22,12 +22,23 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+var (
+	lchownEnabled bool
```

Comment:
If we revert the group of the user, should we remove the flag?

Reply:
We may need it again for testing, I'll leave for now but unused.